### PR TITLE
Added Activation,padding and stride scenarios.

### PR DESCRIPTION
**Pull request by amitsrivastava78**
This is one of the TODO in the file.

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @karimnosseir: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 15 days with no activity and the `awaiting review` label has been applied.

**Comment by amitsrivastava78** [comment]
@karimnosseir , would be great of you can review the PR and provide your feedback.

Regards
Amit

**Comment by karimnosseir** [comment]
Sorry for late reply as i was out of office past few weeks.

File: tensorflow/lite/kernels/depthwise_conv_test.cc [hunk 1]
```diff
@@ -38,8 +38,73 @@ using ::testing::ElementsAreArray;
...
+  BaseDepthwiseConvolutionOpModel(
+      TfLiteRegistration* registration, const TensorData& input,
+      const TensorData& filter, const TensorData& output, Padding padding_type,
+      int stride_width, int stride_height,
+      const ActivationFunctionType& fused_activation_function,
+      int dilation_factor = 1) {
+    input_ = AddInput(input);
+    filter_ = AddInput(filter);
+
+    int bias_size = GetShape(filter_)[3];
```
Comment (jianlijianli) [inline tensorflow/lite/kernels/depthwise_conv_test.cc]: please make this a constant.

File: tensorflow/lite/kernels/depthwise_conv_test.cc [hunk 2]
```diff
@@ -38,8 +38,73 @@ using ::testing::ElementsAreArray;
...
+        // per tensor quantization.
+        auto bias_scale = GetScale(input_) * GetScale(filter_);
+        TensorData bias{TensorType_INT32, {bias_size}, 0, 0, bias_scale};
+        bias_ = AddInput(bias);
+      }
+    }
+
+    output_ = AddOutput(output);
+
+    int input_depth = GetShape(input_)[3];
```
Comment (jianlijianli) [inline tensorflow/lite/kernels/depthwise_conv_test.cc]: please make those constant as well.

**Review by jianlijianli — APPROVED** [review]
Thanks for adding the test case. Just some minor comments.

**Pull request merged**